src/js/strategies/CitrixVDIStrategy.js

```diff
@@ -6,28 +6,41 @@ import CCPInitiationStrategyInterface from "./CCPInitiationStrategyInterface";
 import {hitch} from "../utils";
 import {FailedState} from "../rtc_session";
 import {RTC_ERRORS} from "../rtc_const";
-import "@citrix/ucsdk/CitrixWebRTC";
+
+
 
 export default class CitrixVDIStrategy extends CCPInitiationStrategyInterface {
 
-    constructor() {
+    constructor(useRealCitrix = true) {
         super();
-        if (!window.CitrixWebRTC.isFeatureOn("webrtc1.0")) {
-            throw new Error('Citrix WebRTC redirection feature is NOT supported!');
-        }
-        window.getCitrixWebrtcRedir = function () {
-            const registryValue = Promise.resolve(1);
-            return new Promise(function (resolve, reject) {
-                //retrieve registry value internally
-                registryValue.then((v) => {
-                    resolve(v);
-                }).catch(() => {
-                    reject();
-                })
-            })
+        if(useRealCitrix){
```

Comment:
[Query] 
1. Does `useRealCitrix` intend to import Citrix WebRTC SDK dynamically?
2. After this change, will we see HDX related logs in the console for native CCP?

Reply:
1. Yes
2. with 3.1.0, HDX related log is gone as HDX init would not happen automatically, unless we can initUCSDK. However, based on the unit test behavior, I would assume v2.0.5 will not have HDX related log with this change